### DISCUSSION TITLE
ui.step() "highlight"

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1168] [Q&A] [answered]
In a `ui.stepper()`, is there any way to specify which ui.step() is "highlighted"; i.e. not greyed out?

In the example below, the first `ui.step()` seems to have "focus":
<img width="1324" alt="image" src="https://user-images.githubusercontent.com/64787868/147870267-5e5b76bb-a87d-4e95-b301-fa025647b39c.png">

I have explored the parameter `done`, but it removes the `icon`.

**Comment by mturoci** [comment, marked as answer]
> is there any way to specify which ui.step() is "highlighted"; i.e. not greyed out?

The highlighted step is always the one following the `done` step.

> I have explored the parameter done, but it removes the icon.

That's right, it replaces the icon with a checkmark to indicate the step is already finished (visited) by the user.
